templates/shunting-yard-config.yml.tmpl

```diff
@@ -0,0 +1,19 @@
+---
+# Copyright (C) 2018 Expedia Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+
+# This is a sample file
```

Comment:
Can you remove this comment?  i think this is a `template` file.

Reply:
removed.